src/compressed_tensors/compressors/sparse_compressors/base.py

```diff
@@ -97,8 +111,10 @@ def decompress(
         :param device: device to load decompressed weights onto
         :return: iterator for generating decompressed weights
         """
-        weight_mappings = get_nested_weight_mappings(
-            path_to_model_or_tensors, self.COMPRESSION_PARAM_NAMES
+        weight_mappings, uncompressed_params = get_nested_weight_mappings(
```

Comment:
Why do we need a separate `uncompressed_param`?
What parameter would we have that isn't required to be loaded to decompress the weight?

Reply:
This is needed for cases where all modules have not been targetted, like for example model.embed_tokens, since these weights should also be loaded back into the model

Reply:
Can we call them "ignored_parameters" or something? I thought this was one of the parameters needed for compression/decompression but not necessarily compressed itself (such as the row_offsets).

Reply:
Accepted!